Normalizer/CamelKeysNormalizer.php

```diff
@@ -75,8 +75,17 @@ private function normalizeString($string)
             return $string;
         }
 
-        return preg_replace_callback('/_([a-zA-Z0-9])/', function ($matches) {
+        if (preg_match('/^(_+)(.*)/', $string, $matches)) {
```

Comment:
or should we just ignore anything with a leading underscore? or should we just ignore a single underscore?